WordPress/Classes/ViewRelated/Me/AppSettingsViewController.swift

```diff
@@ -200,7 +200,9 @@ class AppSettingsViewController: UITableViewController {
 
     func toggleGutenberg() -> (Bool) -> Void {
         return { [weak self] _ in
-            GutenbergSettings().toggleGutenberg()
+            let settings = GutenbergSettings()
+            settings.toggleGutenberg()
```

Comment:
I'm curious, is this just a style improvement, or was something not behaving as expected?

Reply:
Right! It was calling the method to sync the setting with remote too (a second call on the same instance) but now that second call was removed so this is not really needed anymore. 👍